src/Services/ExtensionService.cs

```diff
@@ -163,46 +159,49 @@ public async Task<IEnumerable<IExtensionWrapper>> GetInstalledExtensionsAsync(bo
                 var extensions = await GetInstalledAppExtensionsAsync();
                 foreach (var extension in extensions)
                 {
-                    var (devHomeProvider, classId) = await GetDevHomeExtensionPropertiesAsync(extension);
-                    if (devHomeProvider == null || classId == null)
+                    var (devHomeProvider, classIds) = await GetDevHomeExtensionPropertiesAsync(extension);
+                    if (devHomeProvider == null || classIds.Count == 0)

```

Comment:
They shouldn't actually ever be null, I updated the GetDevHomeExtensionPropertiesAsync method to return an empty list instead of returning null for the ClassIds. So we should be good here.